src/include/function/scalar_macro.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <unordered_map>
+
+#include "parser/macro/create_macro.h"
+
+namespace kuzu {
+namespace function {
+
+using parameter_value_map = std::unordered_map<std::string, parser::ParsedExpression*>;
+
+struct ScalarMacro {
```

Comment:
Should we call this ScalarMacroFunction?

Reply:
renamed